Sofascore2025/Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,220 @@
+import UIKit
+import SnapKit
+import SofaAcademic
+
+final class EventView: BaseView {
+    
+    private let event: Event
+
+    public init(event: Event) {
+        self.event = event
+        super.init()
+        configure(with: event)
+    }
+    
+    private let homeTeamImageView = UIImageView()
+    private let awayTeamImageView = UIImageView()
+    private let leftLabel = UILabel()
+    private let timeLabel = UILabel()
+    private let minuteLabel = UILabel()
+    private let homeTeamLabel = UILabel()
+    private let awayTeamLabel = UILabel()
+    private let numHomeTeamGoalsLabel = UILabel()
+    private let numAwayTeamGoalsLabel = UILabel()
+    private let mainView = UIView()
+    private let mainStackView = UIStackView()
+
+    internal override func addViews() {
+        addSubview(mainStackView)
+        mainStackView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        mainView.addSubview(homeTeamImageView)
+        mainView.addSubview(leftLabel)
+        mainView.addSubview(awayTeamImageView)
+        mainView.addSubview(timeLabel)
+        mainView.addSubview(minuteLabel)
+        mainView.addSubview(homeTeamLabel)
+        mainView.addSubview(awayTeamLabel)
+        mainView.addSubview(numHomeTeamGoalsLabel)
+        mainView.addSubview(numAwayTeamGoalsLabel)
+        
+        mainStackView.addArrangedSubview(mainView)
+
+        styleViews()
+    }
+
+    internal override func styleViews() {
+        homeTeamImageView.contentMode = .scaleAspectFit
+        homeTeamImageView.clipsToBounds = true
+
+        awayTeamImageView.contentMode = .scaleAspectFit
+        awayTeamImageView.clipsToBounds = true
+
+        leftLabel.textAlignment = .center
+        leftLabel.font = UIFont.systemFont(ofSize: 16, weight: .bold)
```

Comment:
bilo bi super kad bi napravio ekstenziju nad UIFont-om i u njoj definirao fontove koje koristis tu npr onda bi ovo mogao samo zvat ovako
```suggestion
        leftLabel.font = .headline
```